### PR TITLE
Update release notes draft to version v1.27.0-beta.0

### PR DESCRIPTION
**Pull request by yrs147**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.27.0-beta.0

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

File: releases/release-1.27/release-notes/maps/pr-112334-map.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pr: 112334
+releasenote:
+  text: Update the Event series starting count when emitting isomorphic events from
```
Comment (harshanarayana) [inline releases/release-1.27/release-notes/maps/pr-112334-map.yaml:3]:
```suggestion
  text: Updated the Event series starting count when emitting isomorphic events from
```

File: releases/release-1.27/release-notes/maps/pr-112670-map.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+pr: 112670
+releasenote:
+  text: ' Migrate `pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go, pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go
```
Comment (harshanarayana) [inline releases/release-1.27/release-notes/maps/pr-112670-map.yaml:3]:
```suggestion
  text: ' Migrated `pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go, pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go
```

File: releases/release-1.27/release-notes/maps/pr-113840-map.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pr: 113840
+releasenote:
+  text: ' Migrated the StatefulSet controller (within `kube-controller-manager) to
```
Comment (harshanarayana) [inline releases/release-1.27/release-notes/maps/pr-113840-map.yaml:3]:
```suggestion
  text: ' Migrated the StatefulSet controller (within `kube-controller-manager`) to
```

File: releases/release-1.27/release-notes/maps/pr-113910-map.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+pr: 113910
+releasenote:
+  text: ' Migrated the ClusterRole aggregation controller (within `kube-controller-manager`)
```
Comment (harshanarayana) [inline releases/release-1.27/release-notes/maps/pr-113910-map.yaml:3]:
```suggestion
  text: ' Migrated the `ClusterRole` aggregation controller (within `kube-controller-manager`)
```

File: releases/release-1.27/release-notes/maps/pr-116043-map.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+pr: 116043
+releasenote:
+  text: ' From now on, the HPA controller will return an error for the container resource
```
Comment (harshanarayana) [inline releases/release-1.27/release-notes/maps/pr-116043-map.yaml:3]: Remove leading space after `'`. Seems like a few other files in the PR has the same. Please clean them all up ?

File: releases/release-1.27/release-notes/maps/pr-116119-map.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+pr: 116119
+releasenote:
+  text: |-
+    ```
+    #### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
+
+     <!--
+     This section can be blank if this pull request does not require a release note.
```
Comment (harshanarayana) [inline releases/release-1.27/release-notes/maps/pr-116119-map.yaml:8]: Could be replaced with `None`. Same applied for all other files in this PR with the same text

File: releases/release-1.27/release-notes/maps/pr-116171-map.yaml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+pr: 116171
+releasenote:
+  text: ' '
```
Comment (harshanarayana) [inline releases/release-1.27/release-notes/maps/pr-116171-map.yaml:3]: Please add `None` message to this.

**Review by harshanarayana — CHANGES_REQUESTED** [review]

File: releases/release-1.27/release-notes/maps/pr-116043-map.yaml (hunk 5 above)
Comment (yrs147) [reply releases/release-1.27/release-notes/maps/pr-116043-map.yaml:3]: sure

**Comment by harshanarayana** [comment]
@yrs147  Mind squashing these commits into a single one ?

File: releases/release-1.27/release-notes/maps/pr-110566-map.yaml [hunk 8]
```diff
@@ -0,0 +1,4 @@
+pr: 110566
+releasenote:
+  text: 'Setting the Kubelet config option ``--resolv-conf=Host`` on Windows will
```
Comment (harshanarayana) [inline releases/release-1.27/release-notes/maps/pr-110566-map.yaml:3]: No need for double '``'

**Review by harshanarayana — CHANGES_REQUESTED** [review]

**Comment by yrs147** [comment]
@harshanarayana

**Comment by harshanarayana** [comment]
/hold 

@yashsingh74  please run a `git rebase {upstream branch}` against your local branch and the do a `git rebase -i <comit>` and squash all commits into a one and push changes.

**Comment by yrs147** [comment]
> please run a `git rebase {upstream branch}` against your local branch and the do a `git rebase -i <commit>` and squash all commits into a one and push changes.

@harshanarayana I've made the necessary changes , please take a look at it

**Review by harshanarayana — APPROVED** [review]
/lgtm
/approve

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**